index.html

```diff
@@ -9,39 +9,290 @@
-        </p>
+            <p class="section_body">
+                I’m a software developer! I can help you build a product, feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
 
-        <a href="microverse.org" class="connect">LET’S CONNECT</a>
+            <a href="microverse.org" class="connect">LET’S CONNECT</a>
 
-        <ul class="flex-container item_first">
-             <li><a href="microverse.org" class="item"><img src="img/svg/Vector-5.svg"/></a></li>
-             <li><a href="microverse.org"><img src="img/svg/Vector-6.svg"/></a></li>
-             <li><a href="microverse.org"><img src="img/svg/Vector-7.svg"/></a></li>
-             <li><a href="microverse.org" class="item"><img src="img/svg/Vector-8.svg"/></a></li>
-             <li><a href="microverse.org" class="item"><img src="img/svg/Vector-9.svg"/></a></li>
-             <li><a href="microverse.org" class="item"><img src="img/svg/Vector-10.svg"/></a></li>
-        </ul>
+            <ul class="flex-contain">
+                <li><a href="microverse.org" class="item"><img src="img/svg/Vector-5.svg" /></a></li>
+                <li><a href="microverse.org"><img src="img/svg/Vector-6.svg" /></a></li>
+                <li><a href="microverse.org" class="item"><img src="img/svg/Vector-7.svg" /></a></li>
+                <li><a href="microverse.org" class="item"><img src="img/svg/Vector-8.svg" /></a></li>
+                <li><a href="microverse.org" class="item"><img src="img/svg/Vector-9.svg" /></a></li>
+                <li><a href="microverse.org" class="item"><img src="img/svg/Vector-10.svg" /></a></li>
+            </ul>
 
+        </section>
     </section>
+
+    <section class="gridclass">
+
+        <section class="cardone">
+            <img src="img/Snapshoot Portfolio-2.png" alt="">
+            <div>
+                <h1 class="cardonept">
+                    Tonic
+                </h1>
+                <ul class="frame2">
+                    <li id="listid"> CANOPY </li>
+                    <!-- <img src="img/Counter.png" alt=""> -->
+                    <li class="lineitem">Back End Dev</li>
+                    <!-- <img src="img/Counter.png" alt=""> -->
+                    <li class="lineitem">2015</li>
+                </ul>
+
+                <p class="tagtext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+
+                <ul class="frame2">
+                    <li class="tag">html </li>
+                    <li class="tag">css</li>
+                    <li class="tag">javaScript</li>
+                </ul>
+
+                <div>
+                    <div class="action">
+                        <button class="buttonaction">See Project</button>
+                    </div>
```

Comment:
- [ ] Please make sure the button (e.g., "See project") follow the button styling: Active, Hover and Pressed (see design) :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206780439-691eefe6-17f7-4f10-8434-83ea7f06a7c1.png)
 
Please fix this for all your buttons